### DISCUSSION TITLE
Basic Default Page Template

### DISCUSSION THREAD

**Discussion by nathanaskinner** [flyntwp/flynt#549] [General]
How can I implement a basic page template like a post, using the block editor?

**Comment by aaronmeder** [comment]
Hey @nathanaskinner 

the block editor (Gutenberg) is only used for posts with Flynt.
Most of the site is built by using custom components based on ACF, Timber, Twig and (if needed) Javascript web components (see many examples in the `Components` directory, e.g. the [Grid Image Text](https://github.com/flyntwp/flynt/tree/master/Components/GridImageText) component).

If you'd like to write posts with the block editor you can just go ahead without doing anything else.

If you want to create a "programmatic page template" for a custom post type you can have a look at [single.php](https://github.com/flyntwp/flynt/blob/master/single.php) and create a copy of it (with a copy of the included twig template file) and use that to render some custom components.

Could you try to clarify what you're trying to achieve?

Happy evening!
Aaron
